src/Components/Register.jsx

```diff
@@ -0,0 +1,50 @@
+import { useDispatch } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import { createUser } from '../Redux/Slices/usersSlice';
+
+const user = (userName, nickname) => ({
+  name: userName,
+  username: nickname,
+});
+
+const Register = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const CreateUser = (e) => {
+    const btn = e.target;
+    const dataForm = btn.parentElement.parentElement;
+    const name = dataForm.querySelector('#name').value;
+    const username = dataForm.querySelector('#username').value;
+    const msg = dataForm.querySelector('#message-error');
+    const response = dispatch(createUser(user(name, username)));
+
+    response.then((data) => {
+      if (typeof data.payload !== 'object') {
+        msg.textContent = 'User already exist';
+      } else {
+        msg.textContent = '';
+        navigate('/MainPage');
+      }
+    });
+  };
+
+  return (
+    <section className="image-back d-flex-col">
+      <div className="d-flex-col form-cont gap">
+        <h2 className="title-form">Register</h2>
+        <form className="d-flex-col gap" onSubmit={(ev) => ev.preventDefault()}>
+          <span id="message-error" />
+          <input id="name" type="text" placeholder="Name" className="form-input" />
+          <input id="username" type="text" placeholder="Username" className="form-input" />
```

Comment:
@AndreaM2429 I kindly suggest that you create a validation to alert the users that they cannot submit an empty form and prevent the application from sending a request to the API with empty information.

![Response from the API](https://github.com/HFG43/final_capstone_front_end/assets/131293775/018eb893-44e9-46b4-81c8-312a69007e93)

Reply:
Thanks for the reminder, I did it 🙌